src/test/isolation2/expected/modify_table_data_corrupt_optimizer.out

```diff
@@ -163,26 +162,25 @@ ABORT
 
 -- For orca, this will error out
 explain (costs off) update tab1 set a = 999 from tab2, tab3 where tab1.a = tab2.a and tab1.b = tab3.a;
- QUERY PLAN                                                                                        
----------------------------------------------------------------------------------------------------
- Update                                                                                            
-   ->  Redistribute Motion 3:3  (slice3; segments: 3)                                              
-         Hash Key: tab1.b                                                                          
-         ->  Split                                                                                 
-               ->  Result                                                                          
-                     ->  Hash Join                                                                 
-                           Hash Cond: (tab3.a = tab1.b)                                            
-                           ->  Seq Scan on tab3                                                    
-                           ->  Hash                                                                
-                                 ->  Broadcast Motion 3:3  (slice2; segments: 3)                   
-                                       ->  Hash Join                                               
-                                             Hash Cond: (tab2.a = tab1.a)                          
-                                             ->  Seq Scan on tab2                                  
-                                             ->  Hash                                              
-                                                   ->  Broadcast Motion 3:3  (slice1; segments: 3) 
-                                                         ->  Seq Scan on tab1                      
- Optimizer: Pivotal Optimizer (GPORCA)                                                             
-(17 rows)
+ QUERY PLAN                                                                                  
+---------------------------------------------------------------------------------------------
+ Update                                                                                      
```

Comment:
Can you explain the essence of this test?
Why did the plan change, but execution still results in an error?

Reply:
because in this test we have a tuple that is on the wrong segment. in order to update such a tuple, Explicit Redistribute Motion is required, which is not added by ORCA.